templates/stage-deploy-aarch64.yml

```diff
@@ -0,0 +1,46 @@
+stages:
```

Comment:
Looks like it's just a hardcoded pipeline. It's a template repo and it should contain only parametrized bits while the repos that use the templates would specify those params, like CPython versions.